### PR TITLE
fix svhn label mismatch (label 10 -> label 0)

### PR DESCRIPTION
**Pull request by Fyy10**
Hi, I found that the default label setting of the SVHN dataset gives label 10 to digit 0, while in other datasets of Digit-Five, digit 0 is assigned label 0. I think this PR is a bug fix.

PyTorch also has a similar description in their document about [SVHN dataset](https://pytorch.org/vision/stable/generated/torchvision.datasets.SVHN.html#torchvision.datasets.SVHN):

> [SVHN](http://ufldl.stanford.edu/housenumbers/) Dataset. Note: The SVHN dataset assigns the label 10 to the digit 0. However, in this Dataset, we assign the label 0 to the digit 0 to be compatible with PyTorch loss functions which expect the class labels to be in the range [0, C-1]

### CODE REVIEW

**Comment by KaiyangZhou** [comment]
Thanks! But don't worry. The error is corrected at this line when saving the images https://github.com/KaiyangZhou/Dassl.pytorch/blob/master/datasets/da/digit5.py#L18

**Comment by Fyy10** [comment]
Got it. Great work!
